frontend/app/generateUserDetails.js

```diff
@@ -83,7 +92,7 @@ export function rollDice(JSONSend) {
     JSONSend({
         serverAddress: 'cgi-bin/roll_dice.py',
         jsonObject: {user_id: id},
-        successCallback,
+        successCallback: req1 => successCallback(req1),
```

Comment:
I don’t think this change changes anything – before the shorthand notation was being used to refer to the function `successCallback` above, now you’re creating an anonymous function that calls it instead. Correct me if I’m wrong, but won’t that give the same result?

Reply:
Your right, it will give the same result. For some reason I forgot that successCallback was using a default parameter for the logUpdater and therefore needs no intervention from sendJSON. Thanks for pointing that out